packages/main/test/pages/MultiInput.html

```diff
@@ -424,6 +424,17 @@ <h1 class="sample-container-title">Eventing</h1>
 			document.getElementById("dialog").show();
 		});
 
+		document.getElementById("suggestion-token").addEventListener("ui5-change", function (event) {
+			var text = event.target.value;
+			var token = document.createElement("ui5-token");
+
+			token.setAttribute("text", text);
+			token.setAttribute("slot", "tokens");
```

Comment:
You can use the already created functions => addTokenToMI(createTokenFromText(text), "suggestion-token");